app/controllers/concerns/idv/socure_errors_concern.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Idv
+  module SocureErrorsConcern
+    def errors
+      @presenter = socure_errors_presenter(handle_stored_result)
+    end
+
+    def goto_in_person
```

Comment:
having the controller action inside of the concern feels a bit buried 🤔  i didn't realize this was a controller action until I saw the routes file 😬 

verify_info_controller#update has a pattern worth considering